apps/io-wallet-user-func/src/infra/attestation-service/ios/index.ts

```diff
@@ -65,6 +66,13 @@ export const validateiOSAttestation = (
             }),
           E.toError,
         ),
+        TE.chain((attestationValidationResult) =>
```

Comment:
I see you're using fp-ts here anyway. Why don't we use Either(s) as well?

Reply:
This module is separate from the library which is not written in fp-ts